crates/cheatcodes/src/utils.rs

```diff
@@ -171,6 +172,23 @@ pub(super) fn sign(private_key: &U256, digest: &B256, chain_id: u64) -> Result {
     Ok((sig.v, r_bytes, s_bytes).abi_encode())
 }
 
+pub(super) fn sign_p256(private_key: &U256, digest: &B256, _state: &mut Cheatcodes) -> Result {
+    ensure!(*private_key != U256::ZERO, "private key cannot be 0");
+    let n: U256 = "115792089210356248762697446949407573529996955224135760342422259061068512044369".parse()
```

Comment:
should find a better way to do this. Not ideal in the hot path

Reply:
Is this not `NistP256::ORDER`, just like below?

Reply:
ah missed that, thanks!